internal/runbits/dependencies/changesummary.go

```diff
@@ -33,16 +33,18 @@ func OutputChangeSummary(out output.Outputer, newBuildPlan *buildplan.BuildPlan,
 			v := fmt.Sprintf("%s@%s", a.Name(), a.Version())
 			addedString = append(addedLocale, v)
 			addedLocale = append(addedLocale, fmt.Sprintf("[ACTIONABLE]%s[/RESET]", v))
-		}
-		for _, i := range a.Ingredients {
-			dependencies = append(dependencies, i.RuntimeDependencies(true)...)
-			directDependencies = append(dependencies, i.RuntimeDependencies(false)...)
+
+			for _, i := range a.Ingredients {
+				dependencies = append(dependencies, i.RuntimeDependencies(true)...)
+				directDependencies = append(directDependencies, i.RuntimeDependencies(false)...)
+			}
```

Comment:
Note 2 changes here:

1. We no longer look at dependencies if we're iterating over an addition that isn't what the user requested.
2. Fixed `directDependencies` appending to the `dependencies` slice.